library/src/test/java/com/okta/TestUtils.java

```diff
@@ -139,4 +140,24 @@ public static String getUnsignedIdToken(){
     private static String base64UrlNoPaddingEncode(byte[] data) {
         return Base64.encodeToString(data, Base64.URL_SAFE | Base64.NO_PADDING | Base64.NO_WRAP);
     }
+
+    public static AuthorizationRequest.Builder getMinimalAuthRequestBuilder(String responseType) {
+        return new AuthorizationRequest.Builder(
+                getTestServiceConfig(),
+                TEST_CLIENT_ID,
+                responseType,
+                TEST_APP_REDIRECT_URI);
+    }
+
+    public static AuthorizationRequest.Builder getTestAuthRequestBuilder() {
+        return getMinimalAuthRequestBuilder(ResponseTypeValues.CODE)
+                .setScopes(AuthorizationRequest.Scope.OPENID, AuthorizationRequest.Scope.EMAIL)
+                .setCodeVerifier(TEST_CODE_VERIFIER);
+    }
+
+    public static AuthorizationRequest getTestAuthRequest() {
+        return getTestAuthRequestBuilder().
+                setNonce(null).
```

Comment:
nit: formatting looks a little funny, use `.setNonce()` instead of `. \n setNonce()`

Reply:
done